### DISCUSSION TITLE
Table of contents issue and footer configuration

### DISCUSSION THREAD

**Discussion by Auggie321** [CaiJimmy/hugo-theme-stack#881] [Q&A]
Hi there,
Is there a quick way to configure it to remove or hide the content of the green box ，
and setting the footer to keep the copyright centered at the bottom.

![16966691449755](https://github.com/CaiJimmy/hugo-theme-stack/assets/34760807/4455d681-1583-4736-be60-c9a4e8c58ba6)

**Comment by CaiJimmy** [comment]
> Is there a quick way to configure it to remove or hide the content of the green box 

No, you need to add custom CSS to override it.
